### ISSUE TITLE
Benchmark using hyperfine instead of time

### ISSUE THREAD

**Issue by Nomarian** [jpe90/clp#10]
I just tested, and its true that clp is way faster than everything else. to curb statistical anomalies, use [hyperfine](
https://github.com/sharkdp/hyperfine) instead

(I could do it but I'm on an arm device)

```
hyperfine --warmup 3 --export-markdown tothereadme.md "clp $pathtosqlite3.c" "bat $pathtosqlite3.c" "source-highlight $pathtosqlite3.c"
```

**Comment by jpe90** [comment]
Great idea, I'll see if I can get that up on the readme tonight.

**Comment by jpe90** [comment]
Addressed by e8f84ca611c1be72b97cdeee5ea4797e752856b1
